oci/client.py

```diff
@@ -701,7 +707,10 @@ def tags(self, image_reference: str):
             method='GET'
         )
 
-        return res.json()['tags']
+        return [
```

Comment:
it might be worthwhile to define this conversion in oci/util.py (as a function). while code duplication might not be that important for a trivial str-replacement, it would slightly make code more self-describing, and would allow us to document why it is required at said function.

in addition, I am not quite sure whether OCI-Client is the best place for this, as it is actually slightly too low-level. semver/version-handling is actually done on OCM-level (the oci-client thus far handles content stored in OCI-registries as opaque).